res/skins/LateNight/style.qss

```diff
@@ -684,11 +684,8 @@ QPushButton#pushButtonRecording:checked,
 #RecFeedback[displayValue="2"],
 #BeatgridButtons WPushButton[value="1"],
 #KeyMatchButton[displayValue="1"],
-#KeyUpButton[displayValue="1"], #KeyDownButton[displayValue="1"],
-#HotcueButton[displayValue="1"][highlight="0"],
-#HotcueButton[displayValue="2"][highlight="0"],
-#HotcueButton[displayValue="1"][highlight="1"],
-#HotcueButton[displayValue="2"][highlight="1"] {
+#KeyUpButton[displayValue="1"],
+#KeyDownButton[displayValue="1"] {
   background-color: #db0000;
 }
 
```

Comment:
could you please re-add **blue** for set intro/Outro buttons here? this was wrongly removed in https://github.com/mixxxdj/mixxx/commit/74b098c4c211465844e0c6d17504fc2404a966f0#diff-6761a9e770568e999bffff10ea18bf5c
```
#SpecialCueButton[value="1"] {
  background-color: #0044ff;
}
```